plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -113,6 +113,20 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 				if ( field.hidden ) {
 					return null;
 				}
+
+				const fieldProps = {
+					id: `${ id }-${ field.key }`,
+					errorId: `${ addressType }_${ field.key }`,
+					label: field.required ? field.label : field.optionalLabel,
+					// These may come from core locale settings or the attributes option on custom fields.
+					autoCapitalize: field.autocapitalize,
+					autoComplete: field.autocomplete,
```

Comment:
Should we match up core fields array to also have those in the attributes section?

Reply:
What do you have in mind for this? I feel like this would be a large scope increase for this PR, we can follow it up maybe?

Reply:
Something like this
```php
'email'      => array(
				'label'         => __( 'Email address', 'woocommerce' ),
				'optionalLabel' => __(
					'Email address (optional)',
					'woocommerce'
				),
				'required'      => true,
				'hidden'        => false,
				'index'         => 0,
				'attributes'    => array(
					'autocomplete'   => 'email',
					'autocapitalize' => 'off',
				),
			),
```

So the above custom handling of `autoCapitalize` and `autoComplete` but yeah let's keep it in a follow up to make sure it doesn't touch other stuff, I will create an issue.

Reply:
@senadir I went ahead and created the follow up: https://github.com/woocommerce/woocommerce/issues/43680